### PR TITLE
starship-init: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Continue with the comment https://github.com/tldr-pages/tldr/pull/5901#discussion_r627610995.
I based this page in [gh-completion](https://github.com/tldr-pages/tldr/blob/main/pages/common/gh-completion.md).

### CODE REVIEW

File: pages/common/starship-init.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# starship init
+
+> Print the integration code for the infinitely customizable prompt for any shell.
+> More information: <https://starship.rs>.
+
+- Display the subcommand help:
```
Comment (CleanMachine1) [inline pages/common/starship-init.md:6]:
```suggestion
- Print the subcommand help:
```

The other 2 have print (desc and second command) however this doesn't and

Comment (CleanMachine1) [reply pages/common/starship-init.md:6]: This is getting ridiculous, I have to admit. 
Maybe I (mainly me) should stop changing these unless required, however I feel this one is warranted

File: pages/common/starship-init.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`starship init {{bash|elvish|fish|ion|powershell|tcsh|zsh}}`
+
+- Append the `starship` integration code to `~/.bashrc`:
+
+`starship init bash >> ~/.bashrc`
+
+- Append the `starship` integration code to `~/.zshrc`:
+
+`sarshipgh init zsh >> ~/.zshrc`
```
Comment (bl-ue) [inline pages/common/starship-init.md:20]: Why not token syntax?
```suggestion
- Append the `starship` integration code to `~/.bashrc`:

`starship init {{bash}} >> {{~/.bashrc}}`

- Append the `starship` integration code to `~/.zshrc`:

`starship init {{zsh]} >> {{~/.zshrc}}`
```
(Also, what's `sarshipgh`? 😄)

File: pages/common/starship-init.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# starship init
+
+> Print the integration code for the infinitely customizable prompt for any shell.
```
Comment (bl-ue) [inline pages/common/starship-init.md:3]: Readers of `starship init` will probably already know what `starship` itself is.
```suggestion
> Print shell integration code for starship.
```

File: pages/common/starship-init.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/starship-init.md:6]: 🤔 we use `display` in a lot of pages, `print` is used just in 3 pages.

Comment (CleanMachine1) [reply pages/common/starship-init.md:6]: We'll leave it as display

**Review by CleanMachine1 — APPROVED** [review]
Should be print since the other command uses it and comes under my "print rule" however other than that I approve

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

**Comment by bl-ue** [comment]
Apologies for forgetting to approve 😢

**Pull request merged**